packages/web-wallet/src/components/views/CredentialsList/index.tsx

```diff
@@ -343,7 +337,17 @@ const CredentialsList: FC<Props> = (props: Props): ReactElement => {
           },
         },
         {
-          onSuccess: () => refetchParties().then(() => onCloseImportCredentialModal())
+          onSuccess: async () => {
+            if (correlationId.toLowerCase().startsWith('did:ebsi') && uniformCredential.type.includes('VerifiableAuthorisationToOnboard')) {
+              // We want to call the register in the background, so for now we are not dealing with the result, we just execute the register function
+              void registerDidEbsiOnLedger({
+                did: correlationId,
+                credentialIssuer: issuerName
+              }).catch(() => console.log(`Unable to register ebsi did ${correlationId} for issuer ${issuerName}`))
```

Comment:
Shouldn't we propagate this error?

Reply:
it was stated we want this process to be in the background. i have no ui for unhappy flow. so fow now it just logs. i have no idea what the unhappy path should do